docs/source/guides/ethereum_node.rst

```diff
@@ -0,0 +1,231 @@
+.. _using-eth-node:
+
+======================
+Using an Ethereum Node
+======================
+
+For operations that require an ethereum provider for transaction signing and broadcasting, either a local or remote
+ethereum node may be used. For general background information about choosing a node technology and operation,
+see https://web3py.readthedocs.io/en/stable/node.html.
+
+
+Local Ethereum Node
+~~~~~~~~~~~~~~~~~~~
+
+This is the typical configuration for a locally operated trusted ethereum node. For detailed information on using the
+geth CLI and Javascript console, see https://geth.ethereum.org/interface/Command-Line-Options.
```

Comment:
:+1: - I am happy to see these resources being used here instead of maintaining them ourselves.